Documentation/cheatsheet.rst

```diff
@@ -0,0 +1,137 @@
+#################
+Cilium Cheatsheet
+#################
+
+Below is a short list of examples of the various commands Cilium has to offer.
+If a command you use is missing please don’t hesitate to add it to one of the
+groups or create a new one.
+
+Basics
+======
+
+::
+
+        # Check the status of the agent
+
+        cilium status
+
+::
+
+        # Get the current agent configuration
+
+        cilium config
+
+Policy management
+=================
+
+::
+
+	# Importing a policy
+
+	cilium policy import <my-policy.json>
+
+::
+
+	# Get list of all imported policy rules
+
+	cilium policy get
+
+::
+
+	# Remove all policy
+
+	cilium policy delete --all
+
+Monitoring
+==========
+
+::
+
+	# Monitor cilium datapath notifications
+
+	cilium monitor 
+
+::
+
+	# Verbose output
+
+	cilium monitor -v
+
+::
+
+        # Show notifications only for dropped packet events
+
+        cilium monitor --related-to=<id>
+
+::
+
+	# Filter only the dropped packets
```

Comment:
The description seems to be swapped with the previous command.

Reply:
Does this description match the previous command? Doesn't that just show traffic to/from the endpoint ID passed in?

Reply:
Fixed locally. Will rebase and review before pushing it out. Thanks.